builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -376,6 +375,91 @@ func resourceVSphereVirtualMachine() *schema.Resource {
 	}
 }
 
+func resourceVSphereVirtualMachineUpdate(d *schema.ResourceData, meta interface{}) error {
+	// flag if changes have to be applied
+	hasChanges := false
+	// flag if changes have to be done when powered off
+	rebootRequired := false
+
+	// make config spec
+	configSpec := types.VirtualMachineConfigSpec{}
+
+	if d.HasChange("vcpu") {
+		configSpec.NumCPUs = d.Get("vcpu").(int)
+		hasChanges = true
+	}
+
+	if d.HasChange("memory") {
+		configSpec.MemoryMB = int64(d.Get("memory").(int))
+		hasChanges = true
```

Comment:
Thanks! It's fixed.
Missed that when Iwas splitting the patch (relocation branch an this one).